app/views/layouts/_footer.html.erb

```diff
@@ -31,7 +31,7 @@
             <% end %>
           </div>
           <div class="col-6 footer-links">
-            <h6>Info</h6>
+            <h5><span style="background-color:#42b983; padding:0px; padding-left: 2px; padding-right:2px;">Info</span></h5>
```

Comment:
Can you move the styles to the stylesheet for better readability and maintainability. Also try to use an existing class for the same. @aayush-05 maybe able to help.

Reply:
Thanks for the feedback. I'll surely make the changes and will confirm before making another PR.